src/components/Wall/PendingApplications.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <router-link :to="{name: 'groupApplications'}">
+      <q-card
+        color="blue"
```

Comment:
Black text on blue background seems hard to read. GroupPreviewUI has white text:
![image](https://user-images.githubusercontent.com/4410802/43716601-f76c1e7a-9984-11e8-94fb-13f3e1a1651c.png)